sdk/fixtures/src/main/java/com/klaviyo/fixtures/BaseTest.kt

```diff
@@ -97,6 +97,9 @@ abstract class BaseTest {
     /**
      * Gross way to modify a final static field through reflection
      *
+     * NOTE This will break in future Java versions, but is the only way to mock
```

Comment:
Kind of an unrelated comment, it just came up because I couldn't run unit tests until i fixed which JDK the IDE was using to run my tests.